modules/ROOT/pages/bare-metal.adoc

```diff
@@ -0,0 +1,53 @@
+:experimental:
+= Installing CoreOS on Bare Metal
+
+This guide provides instructions to install Fedora CoreOS to bare metal. Two options are available:
+
+* Installing from ISO
+* Installing from PXE
+
+== Prerequisite
+
+Before installing FCOS, you must have an Ignition configuration file. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+== Installing from ISO
+
+To install FCOS onto bare metal with an ISO, follow these steps:
+
+. Download an ISO image from the https://getfedora.org/coreos/download/[FCOS download page].
++
+NOTE: You can install in either legacy boot (BIOS) mode or in UEFI mode, regardless of what mode the OS will use once installed.
++
+. Burn the ISO to disk and boot it.
+. From the boot menu, press `<TAB>` (isolinux) or `e` (grub) to edit the kernel command line.
+. Add the following parameters to the kernel command line:
+* `coreos.inst.install_dev=/dev/sda`
+* `coreos.inst.stream=testing`
+* `coreos.inst.ignition_url=http://example.com/config.ign`
+* `coreos.inst.platform_id=qemu`
```

Comment:
If users are truly installing to bare metal, I think this should be `coreos.inst.platform_id=metal`

If users are testing ISO install in a VM, then `qemu` should be used.

Reply:
I think both cases should be `metal` - `qemu` means that we're fetching Ignition from the `fw_cfg`, but we aren't doing that here.

Reply:
`coreos.inst.platform_id` is kind of an obscure option.  Unless the user is running the installer to install into a VM, which is an unusual case, it's not needed at all.

Reply:
@bgilbert I.e. we should be able to drop this line entirely and let it default to `metal`, right?